### DISCUSSION TITLE
blog/read-philosophy-text-with-ai/

### DISCUSSION THREAD

**Discussion by giscus** [Pathsis/2750-words#53] [Announcements]
# blog/read-philosophy-text-with-ai/

这篇文章展示了 AI 大模型阅读英文哲学文献的表现。AI 能够准确翻译复杂文本，总结文本要点，并和用户进行深入交流。

https://pathos.page/blog/read-philosophy-text-with-ai/

**Comment by savouer1** [comment]
我也用AI问一些比较小众的问题，大多数Ai都能给出比较合理且完整的答案，但也不排除偶尔的抖机灵，整体来看比传统搜索引擎好用很多，能节省大量的时间。
